### PR TITLE
Tlmblgis 67

### PR DESCRIPTION
**Pull request by gsilver**
for
https://itsjira.umms.med.umich.edu/browse/TLMBLGIS-67

### CODE REVIEW

File: mbofui/Gruntfile.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function(grunt) {
       options: {
         banner: '/*\n <%= pkg.name %> <%= pkg.version %> <%= grunt.template.today("yyyy-mm-dd") %> \n*/\n',
         sourceMap: true,
-        sourceMapName: 'all.min.js.map',
+        sourceMapName: 'app/js/dist/all.min.js.map',
```
Comment (dovek) [inline mbofui/Gruntfile.js:18]: Why is this change necessary?

File: mbofui/app/js/controllers.js [hunk 2]
```diff
@@ -3,6 +3,17 @@
...
 ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter', '$timeout', '$log', 'leafletData', 'Bof' , function($compile, $scope, $rootScope, $filter, $timeout, $log, leafletData, Bof) {
   // setting up init values
+  // get user
+  Bof.GetUser('/api/me/').then(function(userResult) {
+    $rootScope.user = userResult;
+    // only function inmediately invoked - get current events on page load,
+    // it fires after we have a user so that we can determine
+    // which belong to the current user
+    getEvents('/api/events/current/');
+    $rootScope.currentViewUrl = '/api/events/current/';
```
Comment (dovek) [inline mbofui/app/js/controllers.js:13]: Nice. Now if this ever changes we only have to edit it here.

File: mbofui/app/js/factories.js [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
 ocellus.factory('Bof', ['$http', '$log', '$q', '$rootScope', function($http, $log, $q, $rootScope) {
   return {
+    //get user
+    GetUser: function(url) {
+      return $http.get(url).then(
+        function success(result) {
+          return result;
+        },
+        function error(result) {
+          // do some error things
```
Comment (dovek) [inline mbofui/app/js/factories.js:12]: When are we planning on doing proper error handling? Should a JIRA be written for proper error handling?

File: mbofui/Gruntfile.js (hunk 1 above)
Comment (gsilver) [inline mbofui/Gruntfile.js:18]: I was writing out the file to the root of the project. It needs to be in the same space as the concatenated js. I was seeing errors in the logs because of this - they are no longer.

File: mbofui/app/js/factories.js (hunk 3 above)
Comment (gsilver) [inline mbofui/app/js/factories.js:12]: Yep.  I will write the Jira.
